common/src/main/java/org/apache/cassandra/sidecar/common/NodeSettings.java

```diff
@@ -57,23 +82,41 @@ public String partitioner()
         return partitioner;
     }
 
+    @JsonProperty("sidecar")
+    public Map<String, String> sidecar()
+    {
+        return sidecar;
+    }
+
+    public String sidecarVersion()
+    {
+        return sidecar.get(VERSION);
+    }
+
     /**
      * {@inheritDoc}
      */
-    public boolean equals(Object o)
+    public boolean equals(Object other)
```

Comment:
Please do not add those unrelated changes (e.g. unnecessary variable renaming) for future contributions.

Reply:
Acknowledged.